.github/workflows/runner.yml

```
@@ -0,0 +1,34 @@
+name: 🔗 Tiered Pipeline Runner
```

```suggestion
name: 🔗 GHA
```

See https://github.com/godotengine/godot/pull/75338#issuecomment-1484017251 and https://github.com/godotengine/godot/pull/75338#issuecomment-1485166201.